ddtrace/settings/_core.py

```diff
@@ -13,6 +16,17 @@
 FLEET_CONFIG, LOCAL_CONFIG = get_configuration_from_disk()
 
 
+class EnvDD(Env):
+    """Provides support for configuration values from the environment variables and Static files"""
+
+    def __init__(self) -> None:
+        self.fleet_source = FLEET_CONFIG.copy()
+        self.local_source = LOCAL_CONFIG.copy()
+        # Order of precedence: local stable config < environment variables < managed stable config
+        source = {**self.local_source, **os.environ.copy(), **self.local_source}
```

Comment:
```suggestion
        source = ChainMap(self.local_source, os.environ.copy(), self.fleet_source)
```
Why do we need copies here?

Reply:
We don't, I removed this